### PR TITLE
(Docs-Sanity): Convert/Indent tables, remove extra lines in all generic/kafka/openebs/coredns exp

### PR DESCRIPTION
**Pull request by ispeakc0de**
Signed-off-by: shubhamchaudhary <shubham.chaudhary@mayadata.io>

### CODE REVIEW

**Review by ksatchit — DISMISSED** [review]

**Review by chandankumar4 — DISMISSED** [review]

**Review by uditgaurav — DISMISSED** [review]

File: docs/getstarted.md [hunk 1]
```diff
@@ -161,10 +161,10 @@ spec:
   jobCleanUpPolicy: delete
   monitoring: false
```
Comment (uditgaurav) [inline docs/getstarted.md]: jobCleanUpPolicy: 'delete'

File: docs/pod-delete.md [hunk 2]
```diff
@@ -158,9 +158,9 @@ metadata:
...
 spec:
   appinfo:
-    appns: default
+    appns: 'default'
     applabel: 'app=nginx'
-    appkind: deployment
+    appkind: 'deployment'
   # It can be true/false
   annotationCheck: 'true'  
   #ex. values: ns1:name=percona,ns2:run=nginx
```
Comment (uditgaurav) [inline docs/pod-delete.md:166]: should we change `annotationCheck` to `chaosType` ?

File: docs/pod-network-corruption.md [hunk 3]
```diff
@@ -162,7 +162,7 @@ metadata:
   namespace: default
 spec:
   # It can be delete/retain
-  jobCleanUpPolicy: delete
+  jobCleanUpPolicy: 'delete'
   # It can be true/false
   annotationCheck: 'true'
```
Comment (uditgaurav) [inline docs/pod-network-corruption.md:167]: same as above `annotationCheck` to `chaosType` you changed it  for `pod-delete` try to do with every exp

File: docs/pod-delete.md (hunk 2 above)
Comment (ispeakc0de) [reply docs/pod-delete.md:166]: It should be reverse chaosType -> annotationCheck

Comment (uditgaurav) [reply docs/pod-delete.md:166]: Okay, then make it uniform in all the places. Thanks!

**Review by chandankumar4 — APPROVED** [review]

**Pull request merged**